src/main/java/uk/gov/hmcts/reform/waworkflowapi/external/taskservice/TaskClientService.java

```diff
@@ -24,48 +19,23 @@ public TaskClientService(@Autowired CamundaClient camundaClient,
         this.authTokenGenerator = authTokenGenerator;
     }
 
-    @SuppressWarnings("PMD.AvoidLiteralsInIfCondition")
-    public Optional<TaskToCreate> getTask(ServiceDetails serviceDetails, Transition transition) {
-        DmnRequest<GetTaskDmnRequest> requestParameters = new DmnRequest<>(new GetTaskDmnRequest(
-            dmnStringValue(transition.getEventId()),
-            dmnStringValue(transition.getPostState())
-        ));
-
-        List<GetTaskDmnResult> dmnResults = camundaClient.getTask(
+    public void sendMessage(SendMessageRequest sendMessageRequest) {
+        camundaClient.sendMessage(
             authTokenGenerator.generate(),
-            serviceDetails.getJurisdiction(),
-            serviceDetails.getCaseType(),
-            requestParameters
+            sendMessageRequest
         );
-
-        if (dmnResults.isEmpty()) {
-            return Optional.empty();
-        } else if (dmnResults.size() == 1) {
-            String task = dmnResults.get(0).getTaskId().getValue();
-            String group = dmnResults.get(0).getGroup().getValue();
-            DmnValue<Integer> workingDaysAllowed = dmnResults.get(0).getWorkingDaysAllowed();
-            String name = dmnResults.get(0).getName().getValue();
-            return Optional.of((workingDaysAllowed == null)
-                                   ? new TaskToCreate(task, group, name)
-                                   : new TaskToCreate(task, group, workingDaysAllowed.getValue(), name)
-            );
-        }
-        throw new IllegalStateException("Should have exactly one task for transition");
     }
 
-    public void createTask(ServiceDetails serviceDetails, String caseId, TaskToCreate taskToCreate, ZonedDateTime dueDate) {
-        ProcessVariables processVariables = new ProcessVariables(
-            serviceDetails.getJurisdiction(),
-            serviceDetails.getCaseType(),
-            caseId,
-            taskToCreate.getTask(),
-            taskToCreate.getGroup(),
-            dueDate,
-            taskToCreate.getName()
-        );
-        camundaClient.sendMessage(
+    public List<Map<String,DmnValue>> evaluate(EvaluateDmnRequest evaluateDmnRequest, String id) {
```

Comment:
It seems the id is not used here, right?

Reply:
Yh the id is only used for the evaluateDmn endpoint the sendMessage should work via the messageName

Reply:
ok, so then can we remove it? If it's not used...

Reply:
Or maybe you mean it's planned to be used later on.